executers/dependenciesutils_test.go

```diff
@@ -157,6 +159,112 @@ func TestMergeReplaceDependenciesWithGraphDependencies(t *testing.T) {
 	}
 }
 
+func TestParseModForReplaceDependencies(t *testing.T) {
```

Comment:
The test is pretty hard to read as it is now.
Maybe we can move the mod content into a resource file?

Reply:
Done